test/Microsoft.AspNet.JsonPatch.Test/NestedObjectTests.cs

```diff
@@ -1748,5 +1892,10 @@ public void MoveToEndOfListWithSerialization()
             Assert.Equal(0, doc.IntegerValue);
             Assert.Equal(new List<int>() { 1, 2, 3, 5 }, doc.SimpleDTO.IntegerList);
         }
+
+        private void LogErrorMessage(JsonPatchError<SimpleDTOWithNestedDTO> patchError)
+        {
+            ErrorMessage = patchError.ErrorMessage;
+        }
```

Comment:
I typically just avoid statefulness on the test class itself. Your call if you like it better by my or your way :laughing: 

I'd do this like so:

```
public class TestErrorLogger
{
    public string ErrorMessage { get; set; }
}

    private void LogErrorMessage(JsonPatchError<SimpleDTOWithNestedDTO> patchError)
    {
        ErrorMessage = patchError.ErrorMessage;
    }

var logger = new TestErrorLogger();
patchDoc.ApplyTo(doc, logger.LogErrorMessage);

```